README.md

```diff
@@ -1,47 +1,63 @@
-# stsci_rtd_theme
-STScI branded sphinx theme that inherits from sphinx_rtd_theme
+# `stsci-rtd-theme`
 
-This theme is based off of the sphinx_rtd_theme, from which it inherits, but the CSS styling
-has been altered for easier reading on a variety of platforms and browsers.
+This is a Sphinx theme branded for the Space Telescope Sciences Institute (STScI).
```

Comment:
nitpicky bit i dont like the word 'branded' here for some reason - maybe something like 'This is a Sphinx theme for packages maintained by the Space Telescope Sciences Institute (STScI).'